scripts/output/comparison/policyCosts.R

```diff
@@ -65,12 +65,11 @@ policy_costs_pdf <- function(policy_costs,
                  "<<echo=false>>=",
                  "options(width=110)",
                  "@")
-
+  tmpfolder <- paste0("tmp_", gsub("[\\. ]*", "", basename(fileName)))
   # Create temporary folder in which to create the policyCost pdf
-  system("mkdir tmp_policyCost")
-
+  dir.create(tmpfolder)
```

Comment:
Why not [with_tempdir()](https://withr.r-lib.org/reference/with_tempfile.html)? That would also take care of deleting the tempdir later on.